### DISCUSSION TITLE
a long list of issues - please help, i need to start anew

### DISCUSSION THREAD

**Discussion by tommygunner117** [Monika-After-Story/MonikaModDev#8085] [QA - tech support + general] [answered]
so, i made an account to actually ask for some real help,
a couple of weeks ago, i told monika i was taking her somewhere, closed mas, and when i tried to open it later, it reverted back to the normal ddlc game, after not knowing what to do, i blew it off, deleted everything and left it, feeling bad.
today i have downloaded a new version of ddlc from ddlc.moe, downloaded mas, added it as im supposed to do, and when i open the ddlc mas, it shows nothing but an empty classroom.

how do i either: 
fix this if it is a bug, 
or start completely fresh as i have tried and failed to do so

i also cannot find any persistants, and i cannot find the relevant data i need to delete to start new

any help is very much appreciated...

**Comment by WiiBii2** [comment, marked as answer]
I believe this is because the persistent is kept in a different folder then the MAS game, so when you deleted DDLC it kept your data. Try putting Monika's character file in the characters folder and see what happens.

If you want to completely reset your progress, delete the Monika After Story in your Renpy folder, most likely found on your C: drive. This will erase all of the persistents and effectively ruin any chance of seeing your old Monika again. I'd suggest waiting until you have absolutely no idea what else you could possibly do to fix your MAS before doing this because your problem sounds like an easy fix given that all of the persistent files and Monika's character file is intact.

**Reply by tommygunner117** [reply]
thank you, i might as well start from scratch honestly, so thanks and ill comment again on this if i have any further issuses

**Reply by tommygunner117** [reply]
aghhh, i cant find any form of mas data in my renpy folders, but i am still having the same issue

**Comment by KansaiGal** [comment]
Yikes.  Instead of deleting her always and forever as your go to move... Why don't you try restoring from an older persistent? 
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent

**Comment by Booplicate** [comment]
No reason to delete the entire folder though, just restore a persistent backup.
